File: problem_0001/problem_0001.pl
#!/usr/bin/perl 
use strict;
use warnings;

sub is_multiple
{
	my ($n) = @_;

	return 1 if ($n % 3 == 0);
	return 1 if ($n % 5 == 0);
	return 0;
}

my $sum = 0;
for (1..999)
{
	$sum += $_ if (is_multiple($_));
}

print "$sum\n";

File: problem_0001/problem_0001.cpp
#include <iostream>

using namespace std;

int main()
{
  int sum = 0;
  for (int n=0; n<1000; n++){
    if(n%5==0 || n%3==0) sum+=n;
  }
  cout << sum << endl;
  return 0;
}
